lib/zeromq/majordomo/asynchronous_arbiter.rb

```diff
@@ -0,0 +1,49 @@
+require 'ffi-rzmq'
+
+module Zeromq
+  module Majordomo
+    class AsynchronousArbiter
+      INFINITE = -1
+
+      class << self
```

Comment:
Does this have to be a singleton? Can it use a constructor for `timeout` and `md_service` ?